.github/workflows/check-for-reproducer.yml

```diff
@@ -3,9 +3,7 @@ name: Check for reproducer
 # Also, when a comment is added, edited or deleted.
```

Comment:
the comment should be adjusted too.

Maybe keep the issue edited trigger? I think the main noise point comes from the issue comment events